Gemfile.lock

```diff
@@ -198,7 +198,7 @@ GEM
       railties (>= 6.0.0)
       sass-embedded (~> 1.63)
     date (3.4.1)
-    debug (1.10.0)
+    debug (1.11.0)
```

Comment:
It seems some of the railties tests are picking up the framework Gemfile.lock, so we need to bump this to get CI green.